src/Pos/Generator/Block/Param.hs

```diff
@@ -39,16 +42,46 @@ instance Buildable AllSecrets where
             (length _asSecretKeys)
             (HM.keys _asSecretKeys)
 
+-- | Parameters for transactions payload generation.
+data TxGenParams = TxGenParams
+    { _tgpTxCountRange :: !(Word, Word)
+    -- ^ Such (a, d), that there block will include some x ∈ [a, a+d)
+    -- transactions. Set to (y,y) to disable tx generation.
```

Comment:
Hm, `(y, y)` means `[y, y+y)` according to the comment which is not empty as long as `y > 0`. Seems like there is a mistake in this comment.